### PR TITLE
Add Docker Quickstart to installation instructions and remove installer

### PR DESCRIPTION
**Pull request by NicolaiRidani**
## Description

Rewrite of installation instructions (according to [installation tutorial on website](https://www.django-cms.org/en/blog/2020/07/07/5-step-django-cms-tutorial-for-new-developers/)), adding Docker Quickstart and removing django CMS installer

## Related resources

* See project board of documentation workgroup: https://github.com/django-cms/django-cms/projects/6 

## Checklist

* [x] I have opened this pull request against ``develop``
* [x] I have added or modified the tests when changing logic
* [x] I have followed [the conventional commits guidelines](https://www.conventionalcommits.org/) to add meaningful information into the changelog
* [x] I have read the [contribution guidelines ](https://github.com/django-cms/django-cms/blob/develop/CONTRIBUTING.rst) and I have joined #workgroup-pr-review on [Slack](https://www.django-cms.org/slack) to find a “pr review buddy” who is going to review my pull request.

### CODE REVIEW

**Comment by macolo** [comment]
hi @NicolaiRidani great work! I see you have some errors in the automated testing, could you review the error messages and see if you can fix the problems?

**Comment by macolo** [comment]
Also you might need to click on "update branch" or rebase you local branches as there is a warning about "This branch is out-of-date with the base branch"

**Review by marksweb — APPROVED** [review]

File: docs/introduction/01-install.rst [hunk 1]
```diff
@@ -6,127 +6,101 @@
...
+.. image:: /introduction/images/create_page_with_django2_cms.png
+   :alt: create a page with django cms
+   :width: 400
+   :align: center
 
-Start the server using following command
+Here is your newly created page.
 
-::
+*******************************
```
Comment (TiredFingers) [inline docs/introduction/01-install.rst:83]: *********************************

File: docs/introduction/01-install.rst [hunk 2]
```diff
@@ -6,127 +6,101 @@
...
+Publish your first page (Step 4)
```
Comment (TiredFingers) [inline docs/introduction/01-install.rst:84]: Publish your first page (Step 4)

File: docs/introduction/01-install.rst [hunk 3]
```diff
@@ -6,127 +6,101 @@
...
+*******************************
```
Comment (TiredFingers) [inline docs/introduction/01-install.rst:85]: *********************************

**Review by crydotsnake — APPROVED** [review]

**Pull request merged**